### ISSUE TITLE
__meteor_bootstrap__ is not defined

### ISSUE THREAD

**Issue by fracz** [JorgenVatle/meteor-vite#101]
Hi, 
I have just tried to run the example app and it fails with the following error.

```
10:18 $ meteor run
[[[[[ ~/Desktop/simple-todos-vue ]]]]]        

=> Started proxy.                             
=> Started HMR server.                        
=> Errors prevented startup:                  
   
   While loading plugin `vite` from package `jorgenvatle:vite-bundler`:
   packages/meteor.js:725:27: __meteor_bootstrap__ is not defined
   at packages/meteor.js:725:27
   at packages/meteor.js:751:4
   at load (packages/core-runtime.js:139:16)
   at Object.queue (packages/core-runtime.js:134:5)
   at packages/meteor.js:1:8
   
   
=> Your application has errors. Waiting for file change.
=> Started MongoDB.
```

Steps to reproduce:

```
$ meteor create --vue simple-todos-vue
$ cd simple-todos-vue
$ meteor run
```

Meteor 2.14.

**Comment by abate** [comment]
same same over here :cry:

**Comment by SeraVault** [comment]
Same here

**Comment by JorgenVatle** [comment]
Sorry about that. I was a little quick to release the official patch release for Meteor v3 support.

**Comment by JorgenVatle** [comment]
This should be fixed with `jorgenvatle:vite-bundler@v1.12.4`
